src/Umbraco.Infrastructure/Persistence/SqlSyntax/ISqlSyntaxProvider.cs

```diff
@@ -90,6 +90,8 @@ DatabaseType GetUpdatedDatabaseType(DatabaseType current, string? connectionStri
 
     string GetWildcardPlaceholder();
 
+    string GetWildcardConcat(string concatDefault = "");
```

Comment:
I know we don't have them for all methods already, but I think this would benefit from an XML header comment, explaining what it's for.

Reply:
Added.